components/BridgeForm.tsx

```diff
@@ -160,13 +167,76 @@ const BridgeForm: FC<IntrinsicElements["form"] & BridgeFormProps> = ({
 		extension,
 	]);
 
+	const processHistoricalWithdrawRequest = useCallback(async () => {
+		const setTrValue = transferInput.setValue;
+		const transferAmount = Balance.fromInput(
+			transferInput.value,
+			transferAsset
+		);
+		setProgressStatus();
+
+		const eventProof: Awaited<EthEventProof> =
+			await api.derive.ethBridge.eventProof(
+				historicalEventProofId as unknown as EthyEventId
```

Comment:
should this be just `historicalEventProofId as EthyEventId`?

Reply:
webstorm hit me with `TS2352: Conversion of type 'number' to type 'EthyEventId' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.`